src/com/esotericsoftware/kryo/serializers/FieldSerializer.java

```diff
@@ -602,6 +604,12 @@ public void removeField (CachedField removeField) {
 		return fields;
 	}
 
+    /** Get all transient fields controlled by this FieldSerializer
+     * @return all transient  fields controlled by this FieldSerializer */
+	public CachedField[] getTransientFields () {

```

Comment:
Oops, that's true.
That's was temporary measure before I made a pull request of it.
Soon, I will get rid of the public getTransientFields.
Thank you for your advice.